### ISSUE TITLE
listacconts not working

### ISSUE THREAD

**Issue by bobinson** [PBSA/python-peerplays#21]
**Command and the error message:**

```
*[03:23 PM] bobinson@air 🖖  [~]* `peerplays --node wss://ppyws.roelandp.nl/ws  listaccounts`
Traceback (most recent call last):
  File "/Users/b5413b/.pyenv/versions/3.6.5/bin/peerplays", line 11, in <module>
    load_entry_point('peerplays==0.3.7', 'console_scripts', 'peerplays')()
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/b5413b/.pyenv/versions/3.6.5/lib/python3.6/site-packages/peerplays-0.3.7-py3.6.egg/peerplays/cli/decorators.py", line 49, in new_func
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/b5413b/.pyenv/versions/3.6.5/lib/python3.6/site-packages/peerplays-0.3.7-py3.6.egg/peerplays/cli/decorators.py", line 116, in new_func
  File "/Users/b5413b/.local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/b5413b/.pyenv/versions/3.6.5/lib/python3.6/site-packages/peerplays-0.3.7-py3.6.egg/peerplays/cli/wallet.py", line 135, in listaccounts
  File "/Users/b5413b/.pyenv/versions/3.6.5/lib/python3.6/site-packages/graphenelib-1.1.10-py3.6.egg/graphenecommon/wallet.py", line 271, in getAccounts
    accounts.extend(self.getAllAccounts(pubkey))
TypeError: 'DeprecationWarning' object is not iterable
```

**Comment by xeroc** [comment]
Yes, the listaccounts has changed and I haven't implemented the change in peerplays just yet ..

New code can be found here:
https://github.com/bitshares/uptick/blob/fa8714f22e2acee1a8564a06093f78a613754564/uptick/wallet.py#L145

**Comment by bobinson** [comment]
fixed with #26
